platform/ios/src/MGLMapView.mm

```diff
@@ -1566,7 +1604,11 @@ - (void)handleDoubleTapGesture:(UITapGestureRecognizer *)doubleTap
             [self animateWithDelay:MGLAnimationDuration animations:^
              {
                  [weakSelf unrotateIfNeededForGesture];
+                 [weakSelf notifyScaleBarGestureDidEnd];
              }];
+        } else {
+            [self unrotateIfNeededForGesture];
+            [self notifyScaleBarGestureDidEnd];
```

Comment:
I wonder if this will also help with an issue I’m seeing in which pinching multiple times in quick succession causes the scale bar to stay visible; only pinching again gets the scale bar unstuck.